diff --git a/components/pango_geometry/CMakeLists.txt b/components/pango_geometry/CMakeLists.txt
index 281209fa..c0264d92 100644
--- a/components/pango_geometry/CMakeLists.txt
+++ b/components/pango_geometry/CMakeLists.txt
@@ -1,6 +1,6 @@
 get_filename_component(COMPONENT ${CMAKE_CURRENT_LIST_DIR} NAME)
 
-find_package(Eigen3 REQUIRED NO_MODULE)
+find_package(Eigen3 REQUIRED MODULE)
 target_compile_definitions(${COMPONENT} PUBLIC HAVE_EIGEN)
 
 target_sources( ${COMPONENT}
diff --git a/components/pango_image/CMakeLists.txt b/components/pango_image/CMakeLists.txt
index b82e7c3f..59056c43 100644
--- a/components/pango_image/CMakeLists.txt
+++ b/components/pango_image/CMakeLists.txt
@@ -1,6 +1,6 @@
 get_filename_component(COMPONENT ${CMAKE_CURRENT_LIST_DIR} NAME)
 
-find_package(Eigen3 REQUIRED NO_MODULE)
+find_package(Eigen3 REQUIRED)
 target_compile_definitions(${COMPONENT} PUBLIC HAVE_EIGEN)
 
 option(BUILD_PANGOLIN_LIBPNG "Build support for libpng image input" ON)
diff --git a/components/pango_opengl/CMakeLists.txt b/components/pango_opengl/CMakeLists.txt
index 5b8da905..690715ff 100644
--- a/components/pango_opengl/CMakeLists.txt
+++ b/components/pango_opengl/CMakeLists.txt
@@ -22,7 +22,7 @@ PRIVATE
     ${CMAKE_CURRENT_BINARY_DIR}/shaders.cpp
 )
 
-find_package(Eigen3 REQUIRED NO_MODULE)
+find_package(Eigen3 REQUIRED MODULE)
 message(STATUS "Found Eigen: '${EIGEN3_INCLUDE_DIRS}'")
 target_compile_definitions(${COMPONENT} PUBLIC HAVE_EIGEN)
 
